src/components/App.tsx

```diff
@@ -847,6 +878,13 @@ function App() {
               borderRightWidth="1px"
               expand
             >
+              {nodeToRename && (
+                <RenameNodeInput
+                  selectedNode={nodeToRename}
+                  renameNode={renameNode}
+                  setNodeToRename={setNodeToRename}
+                />
+              )}
```

Comment:
since this uses absolute positioning would prefer putting up next to the modal components

Reply:
But if we did that it would appear at the center of the screen, and I think we want this at the center of the left resizable block.

Moving it up the tree as the first child of `Resizable` component, let me know if that works or you want to handle it differently

Reply:
Actually this won't be needed if we handle renaming via editable textbox in custom nodes